backend/auth-service/api/src/main/kotlin/UserDescriptions.kt

```diff
@@ -123,6 +130,26 @@ object UserDescriptions : CallDescriptionContainer("auth.users") {
         }
     }
 
+    val getPrincipal = call<GetPrincipalRequest, GetPrincipalResponse, CommonErrorMessage>("getSecurityPrincipal") {
```

Comment:
Names do not match. Renaming would also be beneficial to match new conventions.
```suggestion
    val retrievePrincipal = call<GetPrincipalRequest, GetPrincipalResponse, CommonErrorMessage>("retrievePrincipal") {
```